test/fuzz/fuzz_doc_symbols.cc

```diff
@@ -0,0 +1,77 @@
+#include "core/Unfreeze.h"
+#include "main/lsp/wrapper.h"
+#include "main/pipeline/pipeline.h"
+#include "payload/payload.h"
```

Comment:
Ditto.

Reply:
I use `sorbet::payload::createInitialGlobalState`. Should I not use that? Or can I get it by including something else?

Reply:
OH. OK, this is fine.